### PR TITLE
Feature/aboutUsSection

### PR DESCRIPTION
**Pull request by jcmonsa**
aboutUsSection component created including the following files:

TEMPLATES:
1. aboutUsSection.hbs
2. personCard.hb

SCSS-COMPONENTS:
1. _aboutussection.scss

STATIC-IMAGES:
1. image 8.png
2. image 8.png
3. image 15.png
4. quotation_close.svg
5. quotation_open.svg

aboutUsSection

### CODE REVIEW

File: templates/aboutUsSection.hbs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="container1">
```
Comment (jrmahechad) [inline templates/aboutUsSection.hbs:1]: I would suggest having a <section> wrapping all the content of this component.
Also you can improve naming on the classes. For example 

```
<section class="aboutus">
    <div class="aboutus__heading">
       <h2 class="about_us__eyebrow">
... and so on
```

File: templates/aboutUsSection.hbs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<div class="container2">
+  <div class="cards-grid__container">
+  {{#each personCard}}
+    {{{module this}}}
+  {{/each}}
+  </div>
+</div>
+
+<div class="quotation-marks__container">
+  <div class="quotation-marks1">
```
Comment (jrmahechad) [inline templates/aboutUsSection.hbs:22]: I would suggest a class name like quotation-marks__open and quotation-marks__close to improve naming!

**Review by jrmahechad — CHANGES_REQUESTED** [review]
Hi @jcmonsa Great job!
I added a couple of comments to improve the structure and naming a little bit.

File: templates/aboutUsSection.hbs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  <div class="person_card__cards_grid">
+  {{#each personCard}}
+    {{{module this}}}
+  {{/each}}
+  </div>
+</section>
+
+<div class="quotation_marks__container">
+  <div class="quotation_marks__open">
+    <svg width="60" height="47" viewBox="0 0 60 47" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (JFozzie) [inline templates/aboutUsSection.hbs:23]: Load the svg file from an img tag

**Review by jrmahechad — APPROVED** [review]

File: templates/aboutUsSection.hbs (hunk 3 above)
Comment (jcmonsa) [reply templates/aboutUsSection.hbs:23]: Hello Jordy, done :)! ty

**Comment by annamarriap** [comment]
Hi @jcmonsa.
In the person_card element you are missing the bottom  shadow, is hard to read without it.

Screen Shot 2022-11-02 at 5 41 03 PM

```
background: linear-gradient(180deg, rgba(0, 16, 48, 0) 0%, #001030 96.87%);
```

**Comment by jsebax** [comment]
@jcmonsa Please add the Hover functionality. Show the corresponding quote for each Card

File: src/scss/components/_aboutussection.scss [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+.person_card__title_image {
+  @include heading-4;
+
+  position: absolute;
+  width: 34.6rem;
+  height: 4rem;
+  top: 38.5rem;
+  text-align: center;
+  color: #fff;
```
Comment (jsebax) [inline src/scss/components/_aboutussection.scss:83]: Please use the variables defined for the colors. In the file `_variables.scss`

File: src/scss/components/_aboutussection.scss [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+.person_card__subtitle_image {
+  @include body-m-reg;
+
+  position: absolute;
+  width: 34.6rem;
+  height: 2.4rem;
+  bottom: 2.4rem;
+  text-align: center;
+  color: #fff;
```
Comment (jsebax) [inline src/scss/components/_aboutussection.scss:94]: Same here

File: src/scss/components/_aboutussection.scss [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+  width: 65.4rem;
+  height: 9.6rem;
+  text-align: center;
+  margin: 0 auto;
+  position: absolute;
+  top: calc(100% + 6rem);
+  left: 50%;
+  transform: translateX(-50%);
+  color: #576684;
```
Comment (jsebax) [inline src/scss/components/_aboutussection.scss:108]: Same here

**Review by jsebax — APPROVED** [review]

**Pull request merged**